pkg/extension/workspace/config/helpers.go

```diff
@@ -15,12 +15,14 @@
 package config
 
 import (
-	"github.com/ghodss/yaml"
+	"sigs.k8s.io/yaml"
```

Comment:
would it be possible to just use https://pkg.go.dev/gopkg.in/yaml.v3 on which k8s one is built? The smaller dep, the better 😄?

Reply:
I tried, but we rely on behavior, and we also need "sigs.k8s.io/yaml" elsewhere for coercing yaml to json to struct. It is a bigger bite than it looks, though I agree desirable.

Reply:
https://github.com/tetratelabs/getenvoy/issues/176

Reply:
OK, thanks for clarifying!